docs/using-fake-timers.md

```diff
@@ -0,0 +1,34 @@
+---
+id: using-fake-timers
+title: Using Fake Timers
+sidebar_label: Using Fake Timers
+---
+
+Using real timers in your tests is less common since they depend on real time
+lapse. To solve this, most testing frameworks offer the option to use fake
+timers in your tests so you won't need to rely on real times.
+
+When using fake timers in your tests, all of the code inside your test uses fake
+timers. The common pattern to setup fake timers is usually within the
+`beforeEach`, here's an example of how to do that using jest:
+
+```js
+beforeEach(() => {
+  jest.useFakeTimers()
+})
+```
+
+When doing so, you'll probably want to restore the timers after your test runs.
```

Comment:
We probably want to explain why e.g. cleanup tasks from 3rd party libraries usually rely on real timers.

Reply:
Sure, I'll add this

Reply:
Also fixed this :) thanks